lib/models/file/middleware/remove-associated.js

```diff
@@ -1,6 +1,14 @@
 async function middleware (file) {
   const FileAssignment = require('../../file-assignment');
   await FileAssignment.find({ file: file._id }).cursor().eachAsync(record => record.remove());
+
+  if (file.correctiveAction) {
+    const correctiveAction = require('../../corrective-action');
```

Comment:
Always import constructors as title upper.
```suggestion
    const CorrectiveAction = require('../../corrective-action');
```